mada.py

```diff
@@ -1,79 +1,93 @@
-import json
-import random
-from eda import eda
-from tqdm import tqdm
 from collections import defaultdict
-from deanony import gen_placa, gen_cpf, gen_nome, gen_valor
-
+import deanonymization as deanony
+from tqdm import tqdm
+import argparse
+import random
+import json
+import util
 random.seed(20211109)
 
-def augment(sentence):
-    return eda(sentence, ["[cpf]", "[placa]", "[valor]"])
-
-def bfs(pflow, intent_sample):
+def dfs (pflow, intent_sample, size, example, rate = 0.93, silence = False, mode = "test"):
```

Comment:
This algorithm is now a Breadth-First Search, not a Depth anymore.

Reply:
Solved.